docs/unified/msi/hcl.md

```diff
@@ -44,20 +44,15 @@ hardware.
         | 12th Gen Intel(R) Core(TM) i5-12400F | v1.1.1 | Dasharo HCL Report |
         | 12th Gen Intel(R) Core(TM) i5-12400F | v1.1.3 | Dasharo HCL Report |
         | 12th Gen Intel(R) Core(TM) i5-12500T | v1.1.0 | Dasharo HCL Report |
-        | 12th Gen Intel(R) Core(TM) i5-12500T | v1.1.0 | Dasharo HCL report |
         | 12th Gen Intel(R) Core(TM) i5-12600 | v1.1.0 | Dasharo HCL Report |
-        | 12th Gen Intel(R) Core(TM) i5-12600 | v1.1.0 | Dasharo HCL report |
         | 12th Gen Intel(R) Core(TM) i5-12600K | v1.0.0 | Dasharo HCL Report |
-        | 12th Gen Intel(R) Core(TM) i5-12600K | v1.0.0 | Dasharo HCL report |
         | 12th Gen Intel(R) Core(TM) i5-12600K | v1.1.0 | Dasharo HCL Report |
-        | 12th Gen Intel(R) Core(TM) i5-12600K | v1.1.0 | Dasharo HCL report |
         | 12th Gen Intel(R) Core(TM) i5-12600K | v1.1.1 | Dasharo HCL Report |
         | 12th Gen Intel(R) Core(TM) i5-12600K | v1.1.1-rc4 | Dasharo HCL Report |
         | 12th Gen Intel(R) Core(TM) i5-12600K | v1.1.2 | Dasharo HCL Report |
         | 12th Gen Intel(R) Core(TM) i5-12600K | v1.1.3 | Dasharo HCL Report |
-        | 12th Gen Intel(R) Core(TM) i7-12700K | v1.0.0 | Dasharo HCL Report |
-        | 12th Gen Intel(R) Core(TM) i7-12700K | v1.0.0 | Dasharo HCL report |
         | 12th Gen Intel(R) Core(TM) i7-12700K | v1.0.0 | [Qubes HCL reports][2] |
+        | 12th Gen Intel(R) Core(TM) i7-12700K | v1.0.0 | Dasharo HCL Report |
```

Comment:
@Stanislaw-bnk, you face these weird, wrong changes because your entries were not sorted. Please note that the line has just moved from two lines above, so there is no change here.